libs/openldap/Makefile

```diff
@@ -95,14 +94,12 @@ TARGET_CFLAGS += $(FPIC) -lpthread \
 	-DURANDOM_DEVICE=\\\"/dev/urandom\\\"
 
 CONFIGURE_ARGS += \
-	--enable-shared \
-	--enable-static \
```

Comment:
This is removed, because is it default? You know, I am trying to find out the reason for "clean-up" configure args.